scripts/irods/test/test_iadmin.py

```diff
@@ -2132,6 +2132,10 @@ def test_moduser_type_rodsgroup(self):
                                    'STDERR', 'SYS_NOT_ALLOWED')
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
 
+    def test_moduser_bad_downgrade(self):
```

Comment:
Let's use more words to describe the test. For example:
```python
def test_downgrade_of_service_account_irods_user_is_not_allowed__issue_NNNN(self):
```
Where `NNNN` should match the issue number.